test-clients/src/main/java/com/hedera/services/legacy/regression/SmartContractCreateContract.java

```diff
@@ -95,28 +96,31 @@
   private static long contractDuration;
 
   public static void main(String args[]) throws Exception {
-    Properties properties = TestHelper.getApplicationProperties();
-    contractDuration = Long.parseLong(properties.getProperty("CONTRACT_DURATION"));
-    host = properties.getProperty("host");
-    port = Integer.parseInt(properties.getProperty("port"));
-    node_account_number = Utilities.getDefaultNodeAccount();
-    node_shard_number = Long.parseLong(properties.getProperty("NODE_REALM_NUMBER"));
-    node_realm_number = Long.parseLong(properties.getProperty("NODE_SHARD_NUMBER"));
-    nodeAccount = AccountID.newBuilder().setAccountNum(node_account_number)
-        .setRealmNum(node_shard_number).setShardNum(node_realm_number).build();
-    localCallGas = Long.parseLong(properties.getProperty("LOCAL_CALL_GAS"));
 
     int numberOfReps = 1;
     if ((args.length) > 0) {
       numberOfReps = Integer.parseInt(args[0]);
     }
     for (int i = 0; i < numberOfReps; i++) {
       SmartContractCreateContract scSs = new SmartContractCreateContract();
-      scSs.demo();
+      scSs.demo(host, nodeAccount);
     }
 
   }
 
+  private void setUp() {
```

Comment:
Just curious, can these not be converted to existing EET suites?

Reply:
These can be, but they are taking more time than usual. We will work on the one by one, but in the mean time we can use this to run them in JRS.